### PR TITLE
Release 2.24.0

### PR DESCRIPTION
**Pull request by leightkt**
### Description

_note: Release 2.24.0 moves the default position of the account center from topRight to bottomRight. To reset your application to topRight, include the following when initializing onboard:_
```typescript
  accountCenter: {
      desktop: {
        enabled: true,
        position: 'topRight'
      },
      mobile: {
        enabled: true,
        position: 'topRight'
      }
    }
```

### **✨ Highlights**

- Enhancement: Account center default position, animation, and triggers #1740 
- feat: add Venly connector to packages #1660
- Update: Bump socket.io-parser from 4.2.1 to 4.2.3 in /packages/demo #1737

#### Docs:

- Fix: Docs - Update platforms Trust wallet is available on within injected docs #1743
- Update: Bump socket.io-parser from 4.2.2 to 4.2.3 in /docs #1738

### **📦 Changes per package**

@web3-onboard/core: 2.20.0
@web3-onboard/react: 2.8.5
@web3-onboard/venly: 2.0.0
@web3-onboard/vue: 2.7.4

### CODE REVIEW

File: packages/core/src/views/account-center/AcctCenterTriggerLarge.svelte [hunk 1]
```diff
@@ -54,13 +56,13 @@
 
   $: defaultChainStyles = getDefaultChainStyles(primaryChain && primaryChain.id)
 
-  function maximize() {
-    updateAccountCenter({ expanded: true })
-  }
+  const accountCenter$ = state
```
Comment (Adamj1232) [inline packages/core/src/views/account-center/AcctCenterTriggerLarge.svelte:59]: Can we remove this?

File: packages/core/src/views/account-center/AcctCenterTriggerSmall.svelte [hunk 2]
```diff
@@ -1,49 +1,58 @@
 <script lang="ts">
+  import { shareReplay, startWith } from 'rxjs/operators'
   import { wallets$ } from '../../streams.js'
   import { updateAccountCenter } from '../../store/actions.js'
```
Comment (Adamj1232) [inline packages/core/src/views/account-center/AcctCenterTriggerSmall.svelte:4]: Looks like we may be able to remove this (running `yarn check-all`)

File: packages/core/src/views/account-center/AccountCenterPanel.svelte [hunk 3]
```diff
@@ -0,0 +1,492 @@
...
+  )
+
+  $: defaultChainStyles = getDefaultChainStyles(
+    connectedChain && connectedChain.id
+  )
+
+  const { position } = state.get().accountCenter
+  const { device } = configuration
+
+  const accountCenter$ = state
```
Comment (Adamj1232) [inline packages/core/src/views/account-center/AccountCenterPanel.svelte:61]: Is this being used? (yarn check-all)

**Review by Adamj1232 — COMMENTED** [review]
Screenshot 2023-06-01 at 11 06 08 AM
A few unused vars/imports noted by `yarn check-all`

**Comment by leightkt** [comment]
I can't run yarn check-all now b/c it fails on the demo b/c of the venly package- but I removed everything listed above

**Comment by leightkt** [comment]
Ok- re-ran yarn check-all and no more unused import or variable hints

**Review by Adamj1232 — APPROVED** [review]
Nice!!

**Pull request merged**